libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -243,9 +243,9 @@ describe('ModalHelper', () => {
           });
 
           const expectSize = (size: ModalSize | undefined) => {
-            expect(ionModal.classList.contains('small')).toBe(size === 'small');
-            expect(ionModal.classList.contains('medium')).toBe(size === 'medium');
-            expect(ionModal.classList.contains('large')).toBe(size === 'large');
+            expect(ionModal.classList.contains('kirby-modal--small')).toBe(size === 'small');
+            expect(ionModal.classList.contains('kirby-modal--medium')).toBe(size === 'medium');
+            expect(ionModal.classList.contains('kirby-modal--large')).toBe(size === 'large');
```

Comment:
Please align with existing naming:
```suggestion
            expect(ionModal.classList.contains('kirby-modal-small')).toBe(size === 'small');
            expect(ionModal.classList.contains('kirby-modal-medium')).toBe(size === 'medium');
            expect(ionModal.classList.contains('kirby-modal-large')).toBe(size === 'large');
```